tools/rosbag_storage/src/buffer.cpp

```diff
@@ -65,7 +66,12 @@ void Buffer::ensureCapacity(uint32_t capacity) {
         capacity_ = capacity;
     else {
         while (capacity_ < capacity)
+        {
+          if (static_cast<uint64_t>(capacity) * 2 >=  std::numeric_limits<uint32_t>::max())
```

Comment:
Nitpick: please remove the second consecutive space after the operator. Also using `>` should be sufficient here.